extensions/amp-consent/0.1/consent-policy-manager.js

```diff
@@ -231,25 +278,24 @@ export class ConsentPolicyManager {
    */
   getConsentStringInfo(policyId) {
     return this.whenPolicyResolved(policyId).then(() => {
-      return this.instances_[policyId].getConsentString();
+      return this.consentString_;
     });
   }
 
   /**
-   * Wait for policy instance to be ready.
+   * Wait for policy instance to be registered.
    * @param {string} policyId
    * @return {!Promise}
    */
   whenPolicyInstanceReady_(policyId) {
```

Comment:
nit: Can we renamed this is registered as well? 😄

Reply:
renamed to `whenPolicyInstanceRegistered_`